tamoc/test/test_ambient.py

```diff
@@ -784,3 +785,68 @@ def test_profile_deeper():
     assert_approx_equal(N, 0.0006146292892002274, significant=6)
 
     ctd.close_nc()
+
+def test_profile_temp_in_C():
+    """
+    create a profile "from scratch", using temperature units in C
+
+    NOTE: this was a issue we found -- the compute_pressure call was assuming K
+    """
+    num_points = 6
+    depth = np.linspace(0, 50, num_points)  # Meters
+    temperatures = (13.0 - np.linspace(
+        0, 5, num_points)) # C -- arbitrary linear profile
+    salinity = 33.5 + np.linspace(0, 0.5, num_points)  # PSU
+
+    # Format data for sending to ambient module Profile class
```

Comment:
Added some unit tests for the pressure profile, so we won't re-introduce that bug :-)